### PR TITLE
Bug 1169535 - [Stingray] Collapse folder when focus move to filter

### PR DESCRIPTION
**Pull request by dwi2**
https://bugzilla.mozilla.org/show_bug.cgi?id=1169535

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c970acb5fef418c5ba6c5a728d6febadcc886045

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8db3e4f11446c3a1840c6fe934b3d5fa0297d105

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4d24e0a938d5824f48062b77dfe45878d2b47895

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c28bd99bdaebdcd9e06c09baf34c6a8b631f8587

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=cdb4098ef0bc5215d77503d60f4ed0aa44b4647b

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=2b9185cb27f78ff1eff8a906aa9548eb74bce0eb

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a0f7c3744996c854352ba18b36eb70caaf6903d4

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a23ff924bdd89bb3cfea3d82b29ef36170949af7

File: tv_apps/smart-home/js/home.js [hunk 1]
```diff
@@ -675,12 +675,15 @@
...
                         !this.cardScrollable.isHovering) {
-        this._cleanFolderScrollable();
+        this.cleanFolderScrollable();
       }
     },
 
-    _cleanFolderScrollable: function() {
-      if (this._focusScrollable === this.folderScrollable) {
+    cleanFolderScrollable: function() {
+      var currentFocusedElement = this.spatialNavigator.getFocusedElement();
```
Comment (rexboy7) [inline tv_apps/smart-home/js/home.js]: can we add an argument called `silent` or `doNotChangeFocus` or something for this? Then we don't need to determine id anymore.

Comment (dwi2) [inline tv_apps/smart-home/js/home.js]: ok

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d8f5de7aa41ee0473e7ffc09bc7067bf25666dfe

**Pull request merged**